tests/lib/external/snapd-testing-tools/utils/log_helper.py

```diff
@@ -0,0 +1,192 @@
+"""
+This library provides a set of functions that can be used to
+process the spread output.
+"""
+
+from __future__ import annotations
+
+import re
+
+# Tasks status
```

Comment:
All of those should be proper enums, depending on the type of the interaction you need either a pure enum, a string enum or something else. A set of free-flying strings defined and used in a random places makes it hard to understand what each function can really process.